internal/jobobject/jobhandle.go

```diff
@@ -0,0 +1,36 @@
+package jobobject
+
+import (
+	"unsafe"
+
+	"github.com/Microsoft/hcsshim/internal/winapi"
+	"golang.org/x/sys/windows"
+)
+
+const jobObjectAllAccess = 0x1F001F
+
+func getWindowsContainerJobName(containerID string) string {
+	return "\\Container_" + containerID
+}
+
+// GetWindowsContainerJobHandle opens the job object of a windows container
+func GetWindowsContainerJobHandle(containerID string) (windows.Handle, error) {
```

Comment:
This should go in a different package. It would be nice for `jobobject` to continue to have no idea what a container is.